themes/stanford_basic/src/scss/components/lockup/_lockup.scss

```diff
@@ -20,6 +20,10 @@
 
   .su-lockup__custom-logo {
     max-width: 300px;
+    @include grid-media('xs') {
+      max-width: -moz-available;
```

Comment:
This isn't the normal way we do these. `max-width: 100%` and the preprocess will do any webkit/moz applications.

I don't see how this helps on very small screens though.

Reply:
The problem is that if you give it a pixel value, it'll overflow the container `div` on screen sizes that are too small to handle the max size.  Whereas if you do with `fill-available` or `100%`, it will respect the bounds of the container even at the smaller sizes.  But using `max-width: 100%` produces identical results as the `fill-available` technique, so I'll change it to use that instead.

![image](https://github.com/SU-SWS/stanford_profile/assets/1263892/d7c13d72-51bb-4126-936e-fddf8f8c9ebb)

instead of 
![image](https://github.com/SU-SWS/stanford_profile/assets/1263892/408c311f-077a-43ef-b275-3eac6e3ddba9)